flow/consent_types.go

```diff
@@ -477,58 +474,27 @@ func (n *OAuth2ConsentRequestOpenIDConnectContext) Value() (driver.Value, error)
 //
 // swagger:model oAuth2LogoutRequest
 type LogoutRequest struct {
-	// Challenge is the identifier of the logout authentication request.
-	ID  string    `json:"challenge" db:"challenge"`
-	NID uuid.UUID `json:"-" db:"nid"`
+	// Challenge is used to retrieve/accept/deny the logout request.
+	Challenge string `json:"challenge" db:"challenge"`
 
-	// Subject is the user for whom the logout was request.
-	Subject string `json:"subject" db:"subject"`
+	// Subject is the user for whom the logout was requested.
+	Subject string `json:"subject,omitempty"`
```

Comment:
I don't think this will ever be empty

```suggestion
	Subject string `json:"subject"`
```

Reply:
done